### PR TITLE
`collapsible-content-button` - Support new text fields

### PR DESCRIPTION
**Pull request by kovsu**
- part of #7886 

## Test URLs

https://github.com/download-directory/download-directory.github.io/issues/141

## Screenshot

https://github.com/user-attachments/assets/cc8c88a8-3e46-4d3b-8f2b-0adefcb6fdf7

### CODE REVIEW

**Review by fregante — APPROVED** [review]
Confirmed to work! Can you also add a separator on its left? I think `table-input` can also be restored on this view (but it needs to stay disabled on the old one.) After that feature is fixed (separate PR) I will remove it from the hotfix list

**Comment by kovsu** [comment]
image

image

File: source/features/collapsible-content-button.tsx [hunk 1]
```diff
@@ -3,15 +3,23 @@ import FoldDownIcon from 'octicons-plain-react/FoldDown';
...
 
 import features from '../feature-manager.js';
 import smartBlockWrap from '../helpers/smart-block-wrap.js';
 import observe from '../helpers/selector-observer.js';
 import {triggerActionBarOverflow} from '../github-helpers/index.js';
 
 function addContentToDetails({delegateTarget}: DelegateEvent<MouseEvent, HTMLButtonElement>): void {
+	const container
+		= delegateTarget.form // TODO: remove after March 2024
+		?? delegateTarget.closest('[data-testid="comment-composer"]')!;
```
[the remark this answers is not in the document]
Comment (fregante) [reply source/features/collapsible-content-button.tsx:16]: Actually not need

**Comment by kovsu** [comment]
https://github.com/user-attachments/assets/fd399beb-ff04-4a51-8d91-aac1180120a5

I find a strange thing. As you can see I went to the issue details page first time, and the button hasn't been added. I need to refresh the page, it adds.

**Comment by fregante** [comment]
> I find a strange thing

Likely related to #6554. Ignore for now, or document these issues in #6554

**Pull request merged**